README.md

```diff
@@ -12,13 +12,66 @@ A user-friendly IRC bouncer.
 
 ## Usage
 
-    go run ./cmd/sojuctl create-user <username> -admin
+### Server side
+
+    go run ./cmd/sojuctl create-user <soju username> -admin
     go run ./cmd/soju -listen irc+insecure://127.0.0.1:6667
 
-Then connect with username `<username>/chat.freenode.net` and join `#soju`.
+### Client side
+
+soju can operate in two different modes: multi connection and single 
+connection.
+
+Even if you plan to use it in only one of these mode, you have to understand
+both to use soju properly. We will cover both modes in this tutorial.
+
+#### Single connection mode
+
+In this mode, 1 connection to a irc server = 1 connection to your soju bouncer.
+
+The easiest and fastest way to use this is to specify the address of the
+server in your username in your client configuration. For example to connect
+to freenode, your username will be: `<soju username>/chat.freenode.net`.
+Also set your soju password in the password field in your client configuration.
+
+This will autoconfigure soju by adding a network to the address 
+`chat.freenode.net` and the autoconnect to it. You will now be able to connect
+to any channel like you would normally do. soju will automatically save and 
+restore the channels you were connected to.
+
+#### Multi connection mode
+
+In this mode, a single connection to your soju bouncer can handle 
+multiple irc connections. You will need to manually configure
+each irc connection using the `BouncerServ`.
+
+Connect to your soju server by specifying your soju username in the
+username field in your client and your password in the password field.
+
+You should now be able to send private messages to the `BouncerServ`.
+You can send it commands to configure soju. Create new networks:
+
+```
+/msg BouncerServ network create -addr chat.freenode.net -name freenode
+/msg BouncerServ network create -addr irc.rizon.net:6697 -name rizon
```

Comment:
The `:6697` part is unnecessary. I'd argue using non-default ports is advanced usage, and users can look up the manual to understand how to do it if they need to.